elastica/memory_block/utils.py

```diff
@@ -121,16 +119,14 @@ def make_block_memory_periodic_boundary_metadata(
     # (without ghost and periodic nodes).
     periodic_boundary_node_idx[1, 2::3] = periodic_boundary_node_idx[0, 0::3] + 2
 
-    periodic_boundary_elems_idx = np.zeros((2, 2 * n_rods), dtype=np.int64)
+    periodic_boundary_elems_idx = np.zeros((2, 2 * n_rods), dtype=np.int32)
     # count ghost elems, first rod does not have a ghost elem at the start, so exclude first rod.
     periodic_boundary_elems_idx[0, 0::2][1:] = 2
     # This is for the first periodic elem at the end
     periodic_boundary_elems_idx[0, 1::2] = 1 + n_elem
     periodic_boundary_elems_idx[0, :] = np.cumsum(periodic_boundary_elems_idx[0, :])
     # Add [0, 1, 2, ..., n_rods] to the periodic boundary elems to accommodate miscounting
-    periodic_boundary_elems_idx[0, :] += np.repeat(
-        np.arange(0, n_rods, dtype=np.int64), 2
-    )
+    periodic_boundary_elems_idx[0, :] += np.repeat(np.arange(n_rods), 2)
```

Comment:
why did you remove `np.int64` here.

Reply:
`np.arange`'s natural typing satisfies, no need for explicit.

Reply:
<img width="538" alt="image" src="https://github.com/GazzolaLab/PyElastica/assets/3798023/3706c64d-b4f3-4154-9d37-a8cbd4f1d662">